### ISSUE TITLE
Links from the app open empty Chrome tab

### ISSUE THREAD

**Issue by tscizzle** [bitpay/copay#7490]
Links in the Bitcoin.com application, like "Buy Bitcoin" or "Learn More" seem like they're supposed to take me somewhere, but then a Chrome window opens and it's blank, with no URL.

I just downloaded the app a couple days ago, and these buttons have always shown that behavior. Is there a setting I need, or might this be a bug?

I'm on macOS High Sierra, downloaded Copay from Bitcoin.com. Application says Bitcoin.com version 4.0.4.

I don't use it in a browser, so not sure how to send a link or how to help reproduce beyond what I've already told, but I believe the above is pretty straightforward.

Let me know if I can provide any more info, thanks!

**Comment by Numline1** [comment]
This is a known issue with current Chrome version(s), links opened from 3rd party apps sometimes spawn an empty window instead of opening a new tab with the URL. Restarting Chrome (holding CMD + Q while in the open Chrome window) temporarily fixes the issue.

**Comment by JDonadio** [comment]
We are BitPay, not Bitcoin.com.

**Comment by tscizzle** [comment]
Correct. I'm reporting a bug in the Copay application (unless it's a Chrome bug, which I couldn't know).

The details about Bitcoin.com are just so you have more details. As someone who answers support and investigates bugs for customers, I know how frustrating it is when people reporting things and provide almost no detail, so I made sure to include as much as I could: where I downloaded from, which version, what I'm seeing, etc.
